client/app/components/packages/feis/attached-documents.hbs

```diff
@@ -0,0 +1,13 @@
+<@form.Section @title="Attached Documents">
+  <p>
+  To complete this submission, attach the complete Final Environmental 
```

Comment:
1) Not sure if this is a typo with all instructions, but here it should say "attach the complete**d**"
2) Nit: Indent line 3 please! 

Thank you!

Reply:
Done